Assets/MixedRealityToolkit.SDK/Experimental/Features/Utilities/TapToPlace/HideTapToPlaceLabel.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Microsoft.MixedReality.Toolkit.Experimental;
+using Microsoft.MixedReality.Toolkit.Experimental.Utilities;
+using Microsoft.MixedReality.Toolkit.Utilities.Solvers;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+public class HideTapToPlaceLabel : MonoBehaviour
+{
+    // Start is called before the first frame update
+
+    private TapToPlace tapToPlace;
+    void Start()
+    {
+        tapToPlace = gameObject.GetComponent<TapToPlace>();
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (tapToPlace.IsBeingPlaced)
+        {
+            var label = gameObject.transform.GetChild(0).gameObject;
+            label.SetActive(false);
```

Comment:
An implication of this Update() function is that just the presence of this MonoBehaviour on an object will inccur a per frame call to heirarchy traversal (per @Troy-Ferrell's comment), and also a call to probably do nothing (i.e. it will call SetActive spuriously).

It would maybe be... more interesting to have tapToPlace actually control the active/inactive state of the label, or if a looser coupling is required, have TapToPlace expose an event that this component can listen to and then disable/enable itself.

That way, there's no need to have another Update() loop.

The main reason I mention this is because this Update loop isn't actually doing anything interesting 99.99999999999999999999999999999% of the time - if it were doing something interesting I think having it might be appropriate.

Reply:
> have TapToPlace expose an event that this component can listen to and then disable/enable itself.

I second adding an event of on beingplaced toggled. Will be useful for other scenarios as well

Reply:
Added two new events `OnPlacingStarted` and `OnPlacingStopped`